### DISCUSSION TITLE
Copy cell content?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2366] [Q&A] [answered]
It might be obvious, but I didn't find a way to copy the content of the current single cell to the clipboard.

The man page mentioend clipbaord operations, buts tarnegly only for complete ros/columns...

I guess I must have overlooked something...

**Comment by anjakefala** [comment, marked as answer]
Yup! `zy` =) `z` operations are often a narrowing to the cell.

**Reply by jwnacnud** [reply]
Is it zy or zY (note the uppercase Y)?

**Reply by anjakefala** [reply]
System clipboard is `zY` and internal VisiData clipboard is `zy` =)

**Reply by jwnacnud** [reply]
Thanks for the expert clarification.

**Reply by halloleo** [reply]
Thanks @anjakefala. It works.

Now, when going back to the help page, I understand what 

    zY gzY     yank (copy) contents of current column for current/selected row(s) to system clipboard means:

* `zY`: Copy current cell
* `gzY`: Copy cells in current column for which the rows are selected.

Is that correct?
